pubspec.yaml

```diff
@@ -84,8 +85,8 @@ flutter:
     - family: Chewy
       fonts:
         - asset: assets/fonts/Chewy-Regular.ttf
-    - family: IRANSansMobile
-      fonts:
-        - asset: assets/fonts/IRANSansMobile.ttf
-        - asset: assets/fonts/IRANSansMobile_Bold.ttf
-          weight: 700
\ No newline at end of file
+    # - family: IRANSansMobile
```

Comment:
Don't change the application's font. Use the project's assets.

revert this change and use `git commit -m [message] --amend` and force push your branch